judge/views/contests.py

```diff
@@ -98,12 +102,15 @@ def get_context_data(self, **kwargs):
                     present.remove(participation.contest)
 
         active.sort(key=attrgetter('end_time'))
+        present.sort(key=attrgetter('end_time'))
```

Comment:
Since we are touching this code anyways, let's add a secondary order to these sorts (since `start_time` and `end_time` are commonly duplicated for e.g junior/senior contests).